libs/ledger-live-common/src/wallet-api/constants.ts

```diff
@@ -47,4 +47,6 @@ export const DEFAULT_MULTIBUY_APP_ID = "multibuy-v2";
 
 export const BUY_SELL_UI_APP_ID = "buy-sell-ui";
 
-export const INTERNAL_APP_IDS = [DEFAULT_MULTIBUY_APP_ID, BUY_SELL_UI_APP_ID];
+export const CARD_APP_ID = "card-program";
```

Comment:
Is this a new manifest that will be added to the manifest-api ?
Because I don't see it in the repo at the moment

Reply:
Yes, web team is working in the manifest